frontend/packages/console-shared/src/components/getting-started/GettingStartedCard.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import {
+  Flex,
+  FlexItem,
+  Text,
+  TextVariants,
+  Button,
+  SimpleList,
+  Skeleton,
+  SimpleListItem,
+} from '@patternfly/react-core';
+import { ArrowRightIcon, ExternalLinkAltIcon } from '@patternfly/react-icons';
+
+import './GettingStartedCard.scss';
+
+export interface GettingStartedLink {
+  key: string;
+  loading?: boolean;
+
+  title?: string;
+
+  external?: boolean;
+  /** Default hyperlink location */
+  href?: string;
```

Comment:
is there a scenario where we might have an `href` but external is `false`. if not, can we not use `href` itself for the checks instead of the external prop?